src/components/LoginPanel.tsx

```diff
@@ -29,7 +29,7 @@ const MiddleDiv = styled.div`
     & input:-webkit-autofill:hover,
     & input:-webkit-autofill:focus,
     & input:-webkit-autofill:active {
-        transition: background-color 99999s ease-in-out 0s;
+        -webkit-box-shadow: 0 0 0 9999px ${(props) => props.theme.main.background[1]} inset !important;
```

Comment:
I'm assuming there's some reason you can't change the background colour instead of just adding the inset box-shadow?

Reply:
Yes it is a Chrome thing

https://developer.mozilla.org/en-US/docs/Web/CSS/:autofill

So in order to get rid of the blue background this is the only way